src/seed/stub/ts.stub

```diff
@@ -1,9 +1,9 @@
 import * as Knex from "knex";
 
-exports.seed = function (knex: Knex): Promise<any> {
+export async function seed(knex: Knex): Promise<any> {
```

Comment:
if it returns promise anyway, why make it async?

Reply:
To allow using await inside the function.

Reply:
Await inside the function is kinda broken, though, not sure we want to encourage it: https://github.com/tgriesser/knex/issues/2581

Reply:
Yes we do :) That is really strange though. It doesn't make sense at all that it works when calling then(), but not with await... Maybe we have missed returning a promise from `.then` somewhere 🤔

Reply:
I'll try writing a failing test for that case.